logsrvd/logsrvd.c

```diff
@@ -1503,6 +1504,17 @@ listener_cb(int fd, int what, void *v)
 	}
 	/* TODO: pause accepting on ENFILE and EMFILE */
     }
+
+    /* set keepalive socket option on socket returned by accept */
+    if (logsrvd_conf_tcp_keepalive()) {
+        int keepalive = 1;
+        if (setsockopt(sock, SOL_SOCKET, SO_KEEPALIVE, &keepalive,
+            sizeof(keepalive)) < 0) {
```

Comment:
Please compare to -1 instead of <0 here.

Reply:
fixed